tests/e2e/pact/resources/test-resources.ts

```diff
@@ -24,13 +25,14 @@ function getProcessCases(searchString: string, operation: string) {
     .filter(e => e.requestFile.operation === operation)
     .map(spec => spec.toJestCase())
 }
-export const processSecondaryCareCommunityAcuteOrderCases = getProcessCases("secondary-care community acute", "send")
-export const processSecondaryCareCommunityAcuteOrderUpdateCases = getProcessCases("secondary-care community acute", "cancel")
-export const processSecondaryCareCommunityRepeatDispensingOrderCases = getProcessCases("secondary-care community repeat-dispensing", "send")
-export const processSecondaryCareCommunityRepeatDispensingOrderUpdateCases = getProcessCases("secondary-care community repeat-dispensing", "cancel")
-export const processSecondaryCareHomecareOrderCases = getProcessCases("secondary-care homecare", "send")
-export const processSecondaryCareHomecareOrderUpdateCases = getProcessCases("secondary-care homecare", "cancel")
-export const processPrimaryCareOrderCases = getProcessCases("primary-care", "send")
-export const processPrimaryCareOrderUpdateCases = getProcessCases("primary-care", "cancel")
+
+export const convertCaseGroups = pactGroups.map(pactGroup => new PactGroupCases(pactGroup, getConvertCases(pactGroup)))
```

Comment:
I think this might be clearer if we created the `PactGroupCases` wrapper inside the `getXCases()` functions

Reply:
Also why are we using `pactGroups` here instead of `getConvertPactGroups()`? Same for the other groups of groups

Reply:
pactGroups vs get..PactGroups returns PactGroup type vs string respectively, the first is used for type safety and grouping by matching description, the second is used for the pact provider name so strips out spaces and formats for readability